docs/references/gcp.md

```
@@ -0,0 +1,141 @@
+# Optional: NGC_API_KEY for NIM coverage.
+
+uv run isvctl test run -f isvctl/configs/providers/gcp/config/vm.yaml
+```
+
+The VM suite exercises 11 subtests end-to-end: launch (with GPU + cloud-init + SSH stability gate), tag verification, serial console output, console-RBAC probe (creates two short-lived probe service accounts + a second probe VM), idempotent stop / start / reboot lifecycle, describe (host OS / driver / CPU / container runtime checks), NIM deploy + inference, teardown of all created resources. Wall-clock is roughly 30–45 minutes on a clean operator environment; capacity stockout in one zone triggers a documented walk to the next zone in the preferred list.
+
+## Supported zones for L4 GPU VM tests
+
+The VM domain's zone-walk prefers GCP zones with observed L4 capacity. The reviewed list (in priority order) is:
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)